### PR TITLE
A0-930 Add support for nominate to cliain

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Infrastructure updated accordingly
- [ ] I have made corresponding changes to the documentation
- [ ] New documentation created
- [ ] Bump `spec_version` and `transaction_version` if relevant
- [ ] Bump `aleph-client` version if relevant

### CODE REVIEW

File: scripts/setup_staking.sh [hunk 1]
```diff
@@ -386,6 +424,41 @@ function run_validator_setup() {
...
+
+  # one token more to cover tx fees for bond and nominate
+  tokens_to_transfer=$((nominator_stake_tokens + 1))
+  info "Transferring ${tokens_to_transfer} tokens from rich's account ${rich_account_id} to nominator ${nominator_account_id}"
+  prompt_if_interactive_mode "Press enter to continue"
+  transfer_tokens "${rich_account_seed}" "${nominator_account_id}" "${tokens_to_transfer}" "${namespace}" "${validator_pod_name}"
+
+  info "Bonding nominator account ${nominator_account_id} with the same account as controller"
+  prompt_if_interactive_mode "Press enter to continue"
+  bond "${nominator_seed}" "${nominator_account_id}" "${nominator_stake_tokens}" "${namespace}" "${validator_pod_name}"
```
[the remark this answers is not in the document]
Comment (Marcin-Radecki) [reply scripts/setup_staking.sh:455]: For sake of this exercise, it was enough assumption :)

**Review by maciejzelaszczyk — APPROVED** [review]

**Comment by maciejzelaszczyk** [comment]
Nicely done! :)

File: scripts/run_nodes.sh [hunk 2]
```diff
@@ -60,7 +60,7 @@ validator_ids_string="${validator_ids_string//${IFS:0:1}/,}"
 
 
 echo "Bootstrapping chain for nodes 0..$((N_VALIDATORS - 1))"
-./target/release/aleph-node bootstrap-chain --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
+# ./target/release/aleph-node bootstrap-chain --base-path "$BASE_PATH" --account-ids "$validator_ids_string" --chain-type local > "$BASE_PATH/chainspec.json"
```
Comment (fbielejec) [inline scripts/run_nodes.sh:63]: why is this commented out?

Comment (Marcin-Radecki) [reply scripts/run_nodes.sh:63]: Bug, thanks for noticing, will revert

Comment (Marcin-Radecki) [reply scripts/run_nodes.sh]: done

**Review by fbielejec — APPROVED** [review]

**Pull request merged**